app/views/referees/build_references/reference_information.html.slim

```diff
@@ -15,11 +15,15 @@ h1.govuk-heading-l = t(".heading", name: @referee.job_application.name)
   = f.govuk_radio_buttons_fieldset :under_investigation do
     = f.govuk_radio_button :under_investigation, "true" do
       p.govuk-hint = t(".must_report")
+      p.govuk-hint = t(".give_investigation_details")
+      = f.govuk_text_area :under_investigation_details, label: nil, rows: 5, max_chars: 200
```

Comment:
Suggestion: Move this `200` max_chars to a constant in the form as done in the EmploymentReference.
(There may be a particular reason for the ReferenceInformationForm not to include this length validation?)